dashboard/src/config.py

```diff
@@ -0,0 +1,48 @@
+"""Common configuration."""
+import configparser
+from urllib.parse import urljoin
+
+
+class Config:
+    """Class representing common configuration."""
+
+    CONFIG_FILE_NAME = 'config.ini'
+
+    def __init__(self):
+        """Read and parse the configuration file."""
+        self.config = configparser.ConfigParser()
+        self.config.read(Config.CONFIG_FILE_NAME)
+
+    def get_sprint(self):
+        """Return name of current sprint."""
+        return self.config.get('sprint', 'number')
+
+    def get_project_url(self):
+        """Return URL to a project page on GitHub."""
+        try:
+            url_prefix = self.config.get('issue_tracker', 'url')
+            project_group = self.config.get('issue_tracker', 'group') + "/"
```

Comment:
Do we need the `+ "/"` ?
Shouldn't the outer `urljoin()` below take care of it ?

Reply:
nope, it will replace the group part instead